core/composer.js

```diff
@@ -168,49 +169,54 @@ export function diffBlueprintToWorkspace(blueprintId) {
 }
 
 export function getSourceInfo(sourceName) {
-  return get("/api/v0/projects/source/info/" + encodeURIComponent(sourceName));
+  return get("/api/v1/projects/source/info/" + encodeURIComponent(sourceName));
 }
 
 export function newSource(source) {
-  return post("/api/v0/projects/source/new", source);
+  return post("/api/v1/projects/source/new", source);
```

Comment:
I think this won't work. v1 source API requires the id key and I didn't find any traces we're setting it in this PR. But maybe I overlooked something.

See: https://github.com/weldr/lorax/blob/master/src/pylorax/api/v1.py#L164

Reply:
Thanks for pointing this out. I've set it back to v0.